simulation/lib/simulation/impl.ex

```diff
@@ -89,22 +89,22 @@ defmodule Simulation.Impl do
     |> calc_net_losses()
   end
   
-  defp get_count(results, state) do
+  def get_count(results, state) do
     Enum.count(results, &(&1 == state))
   end
   
-  defp calc_net_losses(stats) do
+  def calc_net_losses(stats) do
     Map.put(stats, :net_losses, stats.busts + stats.losses - stats.wins)
   end
   
```

Comment:
Want a few bonus points? Implement String.Chars for your structure, so you can just do

~~~ elixir
IO.puts " #{target}\t\t#{stats}"
~~~